client/components/backup-retention-management/info-tooltip/index.tsx

```diff
@@ -0,0 +1,69 @@
+import { Button, Gridicon } from '@automattic/components';
+import { useTranslate } from 'i18n-calypso';
+import * as React from 'react';
```

Comment:
Do we need all of `react` to be imported?

Reply:
You are right! Just fixed it.